jet-enterprise.txt

```diff
@@ -6,9 +6,9 @@ Download_ZIP_URL: https://repository.hazelcast.com/download/jet-enterprise/hazel
 Download_ZIP_Size: 669 MB
 Download_TAR_URL: https://repository.hazelcast.com/download/jet-enterprise/hazelcast-jet-enterprise-4.5.4.tar.gz
 Download_TAR_Size: 669 MB
-Docs_HTML: https://jet-start.sh/docs
+Docs_HTML: https://github.com/hazelcast/hazelcast-jet/tree/master/site/docs
```

Comment:
will that update links in https://hazelcast.com/community-edition-projects/downloads/archives?

Reply:
BTW, EE links are in https://hazelcast.com/get-started/download/#hazelcast-jet